src/apps/views/AppsList/AppsList.tsx

```diff
@@ -186,16 +199,19 @@ export const AppsList: React.FC<AppsListProps> = ({ params }) => {
           2000
         );
       }
+      let newAppInstalled = false;
       activeInstallations.forEach(installation => {
         const item = appsInProgress?.find(app => app.id === installation.id);
         if (!item) {
           removeInstallation(installation.id);
           installedAppNotify(installation.name);
           appsInProgressRefetch();
+          newAppInstalled = true;
```

Comment:
can you explain the logic here - if `!item` then a new app is installed?

Reply:
Yes, because it doesn't have app installation in progress